spinners.json

```diff
@@ -1181,5 +1181,165 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+	"fingerdance": {
+		"interval": 160,
+		"frames": [
+			"☝️",
+			"✌️",
+			"🤘",
+			"🤟",
+			"🖖",
+			"🖐",
+			"✋",
+			"🤚",
+			"👆"
+		]
+	},
+	"fistBump": {
+		"interval": 80,
+		"frames": [
+			"🤜    🤛",
+			"🤜    🤛",
+			"🤜    🤛",
+			" 🤜  🤛",
+			"  🤜🤛",
+			" 🤜✨🤛",
+			"🤜 ✨ 🤛"
+		]
+	},
+	"header": {
+		"interval": 80,
+		"frames": [
+			" 🧑⚽️       🧑",
+			"🧑  ⚽️      🧑",
+			"🧑   ⚽️     🧑",
+			"🧑    ⚽️    🧑",
+			"🧑     ⚽️   🧑",
+			"🧑      ⚽️  🧑",
+			"🧑       ⚽️🧑",
```

Comment:
Constant width.

Can you write a test for that? To ensure that all the frames in each spinner has the same width.

Reply:
Adding a test for constant width could be pretty hard, if not impossible. Look at this small example of the fingerDance spinner:

![Kapture 2020-06-28 at 8 57 29](https://user-images.githubusercontent.com/20396367/85940588-95678100-b91d-11ea-8084-55ed478355ad.gif)

No matter which package or method I use, the number is never the same across all frames, even though visually every frame has the same width.

I used this code to log this btw:

<details>
<summary>code</summary>

```js
const logUpdate = require('log-update');
const cliSpinners = require('.');
const stringWidth = require("string-width");
const stringLength = require("string-length");

const spinner = cliSpinners[process.argv[2] || 'dots'];
let i = 0;

setInterval(() => {
	const {frames} = spinner;
	logUpdate(
		'\n' +
		frames[i = ++i % frames.length] + ' Unicorns' + 
		'\n' +
		'js length:\t\t' + frames[i].length +
		'\n' +
		'"string-width" length:\t' + stringWidth(frames[i]) +
		'\n' + 
		'"string-length" length:\t' + stringLength(frames[i])
	);
}, spinner.interval);
```


</details>

Reply:
> Adding a test for constant width could be pretty hard, if not impossible. Look at this small example of the fingerDance spinner:

If the problem is just with that spinner, we could exclude it from that test.

Reply:
Excluding the spinner from the test would've been somewhat of a solution, although the "fix" I had above doesn't work for every terminal I tested. I've removed the specific emoji causing the issue and added a test, so I think this is resolved as well.